docs/guides/DolphinPay.md

```diff
@@ -1,216 +1,245 @@
-   <br/>
+<br/>
 
    <div style={{textAlign: 'center'}}>
-    <img alt="signer-login" src="/img/guides/signer-login.png" width="50%"/>
+    <img alt="signer-init" src="/img/guides/wallet_connected.png" width="80%"/>
    </div>
+<br/>
+
+### Install [Manta Signer](https://signer.manta.network) and Create a zkAccount
+
+Manta Signer is a native app to turbo-charge zero-knowledge proof generation. Signer is available for:
+   * macOS (version 10.5 or later)
+   * Ubuntu (version 18.04 or 20.04)
+   * Windows (version 10 or later)
+
+You can check the navbar to see if you are connected to signer.
 
-   Go to [Dolphin App](https://app.manta.network/dolphin/transact). You should see that signer is connected in the top right corner of the screen:
 <br/>
 
    <div style={{textAlign: 'center'}}>
-    <img alt="signer-connected" src="/img/guides/signer-connected.png" width="30%"/>
+    <img alt="signer-init" src="/img/guides/signer_connected.png" width="80%"/>
+   </div>
+
+   <div style={{textAlign: 'center'}}>
+    <img alt="signer-init" src="/img/guides/signer_not_connected.png" width="80%"/>
    </div>
-<br/>
 <br/>
 
-### Convert public assets to private assets
+If you don't have signer installed, you can find the download link by clicking on the "Connect Signer" button.
 
-   Select the public address you are sending from, the token you are sending, and the amount you want to send to your shielded address. Press "To Private" to submit:
 <br/>
 
    <div style={{textAlign: 'center'}}>
-    <img alt="to-private" src="/img/guides/to-private.png" width="50%"/>
+    <img alt="signer-init" src="/img/guides/signer_install_modal.png" width="50%"/>
    </div>
+<br/>
+
+Open Manta Signer and follow instructions to create your zkAddress. See our [guide to Manta Signer](https://docs.manta.network/docs/guides/MantaSigner) for more details.
 
-   Finally, polkadot.js will prompt you to approve the transfer, and then publish it to the Dolphin blockchain:
 <br/>
 
    <div style={{textAlign: 'center'}}>
-    <img alt="polkadot-js-sign" src="/img/guides/polkadot-js-sign.png" width="50%"/>
+    <img alt="signer-init" src="/img/guides/signer_setup_page.png" width="50%"/>
    </div>
 <br/>
+
+### Get Testnet Tokens
+
+Join [Manta & Calamari's Discord](https://t.co/5BacMMLSCW), navigate to the `#dolphin-faucet` channel, and type `/gimme`. You should see a faucet options prompt:
+
+<br/>
+   <div style={{textAlign: 'center'}}>
+    <img alt="faucet" src="/img/guides/faucet_bot.png" width="70%"/>
+   </div>
 <br/>
 
-### Send private assets to a shielded address
+Enter your polkadot address and claim some `DOL`. Feel free to claim other testnet tokens like `KSM`, `KAR`, `MOVR`, but you will have to bridge them onto Dolphin (see our guide to bridging [here](https://docs.manta.network/docs/guides/DolphinXCMBridge)). If you want to do private transactions with bridged coins, you will still need `DOL` to pay fees. Note that you can finish this guide with only `DOL`.
```

Comment:
https://docs.manta.network/docs/guides/DolphinXCMBridge
change to 
https://docs.manta.network/docs/guides/DolphinBridge

Reply:
fixed